analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CertificateValidationCheck.cs

```diff
@@ -590,4 +591,75 @@ public static void RestoreCertificateValidation(RemoteCertificateValidationCallb
             ServicePointManager.ServerCertificateValidationCallback = prevValidator;
         }
     }
+
+    public class SomeClass
```

Comment:
This is quite a long example. Can't we simplify the test case to make the problem source more obvious?